cogs/ListUsers.py

```diff
@@ -0,0 +1,61 @@
+import discord
+from discord import app_commands
+from discord.ext import commands
+import sqlite3
+import os
+
+class ListUsersCog(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @app_commands.command(
+        name='list_users',
+        description='Lists registered Twitter usernames and their associated channels'
+    )
+    @commands.has_permissions(administrator=True)
```

Comment:
Bug: Invalid permission check. This decorator only applies to prefix commands and does not apply to slash commands.
To enable permission checking:
```py
from src.permission_check import is_administrator
```
and use `@is_administrator()` as a decorator.